tests/011-test-autoextend-by-amount.sh

```diff
@@ -0,0 +1,120 @@
+source $SRCDIR/libtest.sh
+
+cleanup_lvm_thin_device() {
+  local device_mount=$1
+  local thin_device=$2
+
+  umount $device_mount
+  rm -rf $device_mount
+  lvremove -f $thin_device >> $LOGS 2<&1
+}
+  
+# Test if the docker-pool is auto-extending, and by the
+# right amount, which by default is 20% of the LV size.
+test_auto_extend_by_amount() {
+
+  local devs=$TEST_DEVS dev
+  local test_status
+  local testname=`basename "$0"`
+  local vg_name
+  local vg_name="dss-test-foo"
+  local pool_name="docker-pool"
+  local test_status
+  local threshold=60
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  for vg in $(vgs --noheadings -o vg_name); do
+    if [ "$vg" == "$vg_name" ]; then
+      echo "ERROR: $testname: Volume group $vg_name already exists."
+      return 1
+    fi
+  done
+  
+  # Error out if any pre-existing directory WORKDIR/test-thin-dev 
+  if [ -d $WORKDIR/test-thin-dev ]; then
+    echo "ERROR: $testname: Directory $WORKDIR/test-thin-dev already exists."
+    return 1
```

Comment:
It was going to be the place where I mounted the device, but will remove since I can just write to the block.